server/lib/config/pbench-server-default.cfg

```diff
@@ -20,6 +20,14 @@ lock-dir = %(lib-dir)s/locks
 ###########################################################################
 ## Deployment section
 ###########################################################################
+[put-server]
+pbench_web_server = pbench.example.com
+token = Authorisation-token
+api_version = 1
+rest_endpoint = api/v%(api_version)s
+server_rest_url = http://%(pbench_web_server)s/%(rest_endpoint)s
```

Comment:
Given that you're putting the "passthrough server" configuration defaults into `pbench-server-default.cfg` I don't see any point in overriding them in a new `pbench-put-shim.cfg` rather than the normal `pbench-server.cfg`. That's just adding complexity with no value.

Reply:
I thought of keeping the put-shim config file separate from the others. So, that it will be easier to change URLs and tokens.

Reply:
You added the section to a `[put-server]` section in `pbench-server-default.cfg`, so it's natural to override the default in `pbench-server.cfg` just like the other options; it seems to me that introducing a third file to the usual set just makes it easier to lose track of where things are defined.

Reply:
As I said above, I don't think we should be enshrining shim support in the core code (or config files).  When it's deployed, I have no problem with the shim pulling this information from here (and, I like that it's in a separate section), but, in my opinion, these changes should not be merged into the repo.  I understand that they are necessary for testing, and I understand that they will be needed for deployment, but they won't be needed except in those two very specific cases, so I don't think they need to be entered into the defaults or the repo.

To the extent that the unit tests need this information, they can generate or supply it themselves.  For development, you can just edit your own config file.  When we deploy this, we'll do the same.  So, it doesn't need to go into the repo in this form.

Reply:
ok. I will just get rid of the changes from the pbench-server-default.cfg file and leave put-shim.cfg file as it is for now.